.github/workflows/check_for_redirects.yml

```diff
@@ -1,60 +1,37 @@
 name: Redirects Workflow
-on:
-  pull_request:
-    branches: [main]
-    types: [opened, synchronize, labeled]
```

Comment:
Why did we remove this condition?

Reply:
I think `pull_request` runs on `opened` and `synchronize` already and I decided not to use the `labeled` workflow because I didn't want to "fail" the status check anymore. It was confusing people when the workflow failing and I think having the label should be good enough to let our team know that we might need to check for redirects. Open to any feedback on this though!

Reply:
Makes sense! What about `branches: [main]` though? Do maintainers really have to look at PRs that are not targeting `main`?